### PR TITLE
[DOCS] Add Image Rendering

### PR DESCRIPTION
**Pull request by gilbertsoft**
The rendered html version is available here: https://cfa94cdf.ngrok.io/Configuration/ImageRendering/Index.html

### CODE REVIEW

File: Documentation/Configuration/ImageRendering/Index.rst [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+Image Variants
+==============
+
+Bootstrap Package defines this variants by default:
+
+* default
+
+   * breakpoint: 1200px
+   * width: 1100px
```
Comment (medarob) [inline Documentation/Configuration/ImageRendering/Index.rst]: So, this means after the breakpoint 1200px is reached, the max image width is 1100px ? Between 992px - 1200px the max image width is 920px.

File: Documentation/Configuration/ImageRendering/Index.rst [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+| Property      | Data Type | Description                                     |
++===============+===========+=================================================+
+| multiplier    | integer   | The width is multiplied with this value         |
++---------------+-----------+-------------------------------------------------+
+| gutters       | integer   | This value is added to the width before         |
+|               |           | applying the multiplier and substracted         |
+|               |           | afterwards                                      |
++---------------+-----------+-------------------------------------------------+
+| corrections   | integer   | This value is added as last step to the width   |
+|               |           | after all other calculations                    |
```
Comment (medarob) [inline Documentation/Configuration/ImageRendering/Index.rst]: Why do I need this?
What are the changes in the FE if I use all those options?

File: Documentation/Configuration/ImageRendering/Index.rst (hunk 1 above)
Comment (gilbertsoft) [reply Documentation/Configuration/ImageRendering/Index.rst]: I added this example to the docu now

File: Documentation/Configuration/ImageRendering/Index.rst (hunk 2 above)
Comment (gilbertsoft) [reply Documentation/Configuration/ImageRendering/Index.rst]: This explanation and examples are added to the docu now

**Comment by gilbertsoft** [comment]
The rendered html version is available here: https://cfa94cdf.ngrok.io/Configuration/ImageRendering/Index.html

**Pull request merged**